cloudbuild.yaml

```diff
@@ -2,9 +2,6 @@ steps:
   - name: 'node:13'
     entrypoint: yarn
```

Comment:
can be removed as well, docker is doing yarn install themself

Reply:
Cool, I'll change it